app/networkapi/settings.py

```diff
@@ -206,6 +207,14 @@
     CORS_ORIGIN_WHITELIST = env('CORS_WHITELIST')
     CORS_ORIGIN_REGEX_WHITELIST = env('CORS_REGEX_WHITELIST')
 
+# Jenkins
+USE_JENKINS = env('USE_JENKINS')
+
+if USE_JENKINS is True:
```

Comment:
Your problem is here. If you just do `env('USE_JENKINS')`, without telling the `env` object what kind of type it should parse that env var as. By default it always uses strings. Hence, `'True'` != `True`. You probably want to do two things here:
1. remove the `is True` part since it's not needed
2. Add it to this config https://github.com/mozilla/network-api/pull/76/files#diff-5ef1e82173c7bb267f5213fb2497fe71R21 and tell it that it is a bool.